src/controllers/security/index.js

```diff
@@ -56,6 +56,9 @@ class SecurityController extends BaseController {
       action: 'createOrReplaceRole'
     };
 
+    if (options.force) {
+      request.force = true;
+    }
```

Comment:
(nitpicking) can be included directly in the request object just above:

```
    const request = {
      _id,
      body,
      action: 'createRole',
      force: options.force
    };
```

(same for the other two)